app/Services/ImageStorage.php

```diff
@@ -45,7 +45,7 @@ public function put(string $signupId, File $file)
         $contents = file_get_contents($file->getPathname());
 
         // Make sure we're only uploading valid image types
-        if (!in_array($extension, ['jpeg', 'png', 'gif'])) {
+        if (!in_array($extension, ['jpeg', 'jpg', 'png', 'gif'])) {
```

Comment:
After the composer update, this was coming through as `jpg` in our tests instead of `jpeg` & throwing a validation error. I've added this alternate extension to the list of approved types to get things back in business!